src/neptune/common/hardware/gpu/gpu_monitor.py

```diff
@@ -43,6 +45,24 @@ def get_card_usage_percent(self, card_index):
     def get_card_used_memory_in_bytes(self, card_index):
         return self.__nvml_get_or_else(lambda: nvmlDeviceGetMemoryInfo(nvmlDeviceGetHandleByIndex(card_index)).used)
 
+    def get_card_power_usage(self, card_index):
+        return self.__nvml_get_or_else(
+            lambda: int(nvmlDeviceGetPowerUsage(nvmlDeviceGetHandleByIndex(card_index)) // WATTS_TO_MILLIWATTS)
```

Comment:
Could you please move the units changing to the Gauge-level (made it a part of `GpuPowerGauge`) for consistency.